sympy/physics/quantum/tensorproduct.py

```diff
@@ -18,6 +18,8 @@
     matrix_tensor_product
 )
 
+from sympy import srepr
```

Comment:
This seems to be unused.

Reply:
Ah, sorry. Needed it for debugging. Will remove.